src/libraries/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -1430,11 +1435,18 @@ protected override void Dispose(bool disposing)
                 base.Dispose(disposing);
             }
 
-            public override int Read(byte[] buffer, int offset, int count)
+            public unsafe override int Read(byte[] buffer, int offset, int count)
```

Comment:
Is `unsafe` needed here?

Reply:
It's a left over from first attempt. I'll remove it.